doc_style/doc_home.css

```diff
@@ -0,0 +1,84 @@
+.home-container {
+  display: flex;
+  flex-direction: column;
+}
+.home-subheader {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 1rem;
+}
+.home-subheader > * {
+  padding: 1rem;
+}
+.home-hero-img-container {
+  height: 400px;
+}
+.home-hero-img {
+  height: 100%;
+  object-fit: cover;
+}
+.home-text {
+  padding: 2rem;
+}
+.navigation {
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+}
+.nav-logo-container {
+  width: 6rem;
+  display: flex;
+  align-items: center;
+  padding-left: 0.5rem;
+}
+.nav-search {
+  width: 40rem;
+}
+@media screen and (max-width: 1000px) {
+  .nav-search {
+    width: 25rem;
+  }
+}
+@media screen and (max-width: 800px) {
+  .home-cards {
+    flex-direction: column;
+  }
+  .home-card-container {
+    flex-direction: column;
+  }
```

Comment:
You can write this like
.home-cards, 
.home-cards-container {
  flex-direction: column;
  }

Reply:
It is also fixed now